server/game/cards/07.1-DDeeds/PersonalOrnithopter.js

```diff
@@ -0,0 +1,37 @@
+const GameActions = require('../../GameActions/index.js');
+const GoodsCard = require('../../goodscard.js');
+
+class PersonalOrnithopter extends GoodsCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Repeat Shootout: Join Posse',
+            playType: 'shootout:join',
+            cost: ability.costs.payGhostRock(1),
+            condition: () => this.game.shootout,
+            actionContext: { card: this.parent, gameAction: 'joinPosse'},
+            repeatable: true,
+            message: context =>
+                this.game.addMessage('{0} uses {1} to have {2} soar into their posse.', context.player, this, this.parent),
+            handler: () => {
+                this.game.resolveGameAction(GameActions.joinPosse({ card: this.parent }));
+            }
+        });
+
+        this.action({
+            title: 'Shootout: Flee',
+            playType: 'shootout',
+            cost: ability.costs.bootSelf(),
+            message: context => this.game.addMessage('{0} uses {1} to send {2} home booted', context.player, this, this.parent),
+            handler: context => {
+                this.game.shootout.sendHome(this.parent, context);
+                if(this.game.shootout.checkEndCondition()) {
+                    this.game.shootout.cancelStep();
+                }
```

Comment:
Good thinking, but this is not needed as the end condition is implicitly checked after each play during the shootout plays and in case one posse is empty, it will end the shootout.